src/main/resources/scripts/furnace automatisation.sk

```diff
@@ -15,14 +15,15 @@ options:
 	radius: 1
 
 on fuel burn:
-	fuel slot of block will be empty
+	fuel slot of event-block will be empty
+	wait 1 second

```

Comment:
If this doesn't work without a delay, you should just open an issue... It is not ok to circumvent bugs in ugly ways in example scripts.

Reply:
I forgot to remove the one second wait, it doesn't work with or without it.